css/style.css

```diff
@@ -41,19 +38,222 @@ Developer @GA in NYC
 /******************************************
 /* BASE STYLES
 /*******************************************/
-
 body {
+ color: #606161;
+ font-size: 24px;
+ line-height: 1.4;
+ font-family: 'Open Sans', sans-serif;
+ font-weight: 300;
+}
+button {
+  border-radius: 5px;
+  padding: 8px 35px;
+  text-align: center;
+  margin: 10px;
+}
+hr {
+  width: 100%;
+  border: 0;
+  height: 1px;
+  background: #033048;
+  margin-bottom: 30px;
+}
+.navigation {
+  background-color: #033048;
+}
+footer {
+  background: #121212;
+  color: #fff;
 }
-
 /******************************************
 /* LAYOUT
 /*******************************************/
-
 #container {
+  border-box: box-sizing;
   width: 960px;
   margin: auto;
+  background: #f0efef;
+
+}
+/*.left-gutter {
+  width: 10px;
+  float: left;
+  margin-right: 10px;
+}
+.right-gutter { 
+  width: 10px;
+  float: left;
+  margin-left: 10px;
+}*/
+.logo {
+  border-box: box-sizing;
```

Comment:
You've got the property and value reversed for this declaration. Also, you can just declare a border-box reset for all of your HTML at the top of your CSS file - see here - https://css-tricks.com/box-sizing/